socketshark/subscription.py

```diff
@@ -78,6 +81,38 @@ def prepare_service_data(self):
         return await self.perform_service_request('on_unsubscribe',
                                                   raise_error=False)
 
+    def filter_message(self, data):
```

Comment:
I'd name it so something `should_drop_message` as it now returns `True` to mean "drop" message, while python's `filter` drops items when predicate evaluates to a falsy value